packages/widget_toolkit_qr/lib/src/lib_qr_scanner/views/qr_scanner_component.dart

```diff
@@ -36,18 +35,37 @@ class QrScannerComponent<T> extends StatelessWidget {
   final QrScannerConfiguration? cameraPermissionBottomSheetConfiguration;
   final double? spaceBetweenScannerAndLoadingWidget;
 
+  @override
+  State<QrScannerComponent<T>> createState() => _QrScannerComponentState<T>();
+}
+
+class _QrScannerComponentState<T> extends State<QrScannerComponent<T>> {
+  late final MobileScannerController controller;
+
+  @override
+  void initState() {
+    controller = MobileScannerController(formats: [BarcodeFormat.qrCode]);
```

Comment:
We could allow for other options such as the camera toggling features, flash (torch), detection speed or detection timeout... Do you think it would be beneficial?

Reply:
I think that it would be nice to have this feature, but I believe these are outside the scope of this ticket. All of these could be ways to further improve `widget_toolkit_qr`, but I think these feature design decisions should be further discussed

Reply:
Sounds like a nice addition for sure. But agreed, we can discuss with the team first.